crates/store/re_types/definitions/rerun/archetypes/asset3d.fbs

```diff
@@ -33,4 +33,12 @@ table Asset3D (
   /// If omitted, the viewer will try to guess from the data blob.
   /// If it cannot guess, it won't be able to render the asset.
   media_type: rerun.components.MediaType  ("attr.rerun.component_recommended", nullable, order: 2000);
+
+  // --- Optional ---
+
+  /// An optional color for each vertex.
+  vertex_colors: [rerun.components.Color] ("attr.rerun.component_optional", nullable, order: 3100);
+
+  /// A color multiplier applied to the whole asset.
+  albedo_factor: rerun.components.AlbedoFactor ("attr.rerun.component_optional", nullable, order: 3200);
```

Comment:
Needs to specify interaction with existing materials

Reply:
the meshes defined by a complex format like glb/gltf may already have material properties very similar to this. So this should say something about how this combines with existing colors that may already be set

Reply:
yes! I think I'll slightly adjust the formulation, but that is a definition we can work with (:
I believe I would have gone with multiplying with the existing, but that that's a choice 🤷. Arguably overriding the the more powerful tool 👍

Reply:
My formulation wasn't very pretty yes :smiling_face_with_tear:

Reply:
My formulation wasn't very pretty yes :smiling_face_with_tear: